examples/seismic/source.py

```diff
@@ -266,3 +266,47 @@ def wavelet(self, f0, t):
         tcut = 1.5 / agauss
         s = (t-tcut) * agauss
         return np.exp(-2*s**2) * np.cos(2 * np.pi * s)
+
+
+class DGaussSource(WaveletSource):
+    """
+    Symbolic object that encapsulate a set of sources with a
+    pre-defined 1st derivative wavelet of a Gaussian Source:
+
+    Note: For visualizing the second or third order derivative of Gaussian wavelets,
+    the convention is to use the negative of the normalized derivative. In the case
+    of the second derivative, scaling by -1 produces a wavelet with its main
+    lobe in the positive y direction. This scaling also makes the Gaussian wavelet
+    resemble the Mexican hat, or Ricker, wavelet.
+    The validity of the wavelet is not affected by the -1 scaling factor.
+
+    returns the 1st order derivative of the Gaussian wavelet
+    
+    Parameters
+    ----------
+
+    name: str
+        Name for the resulting symbol
+    grid: :class:`Grid`
+        object defining the computational domain.
+    f0: float
+        Peak frequency for wavelet in kHz
+    time: TimeAxis
+        Discretized values of time in ms
+    """
+
+    def wavelet(self, f0, t, a):
+        """
+        Defines the 1st derivative of a Gaussian wavelet with a peak frequency f0
+        at time t.
+        
+        Parameters
+        ----------
+        f0: float
+            Peak frequency in kHz
+        t: TimeAxis
+            Discretized values of time in ms
+        a: float
+            Maximum amplitude, real constant defined by user
```

Comment:
full stop

Reply:
Drop "real constant defined by the user"

Reply:
OK